<launch>
    <node pkg="mpc_ros" type="path_generator" name="path_generator" output="screen">
        <param name="frame_id"   type="string"   value="map"/>
        <param name="radius"    type="double"   value="5.0"/>
    </node>
    <!-- <node pkg="mpc_ros" type="mpc_node" name="mpc_node" output="screen">
        <param name="weight_x"  type="double"   value="10" />
        <param name="weight_y"  type="double"   value="10" />
        <param name="weight_q"  type="double"   value="5" />
        <param name="weight_vx" type="double"   value="0.5" />
        <param name="weight_vy" type="double"   value="0.5" />
        <param name="weight_w"  type="double"   value="0.05" />
    </node> -->
</launch>